From: Shengjing Zhu <zhsj@debian.org>
Date: Fri, 16 Dec 2022 03:12:53 +0800
Subject: Skip flaky test

GoString may return `<nil>`, which is not regonized by go's ParseExpr.
---
 test/stdtypes/stdtypespb_test.go | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/test/stdtypes/stdtypespb_test.go b/test/stdtypes/stdtypespb_test.go
index 7002354..aace58a 100644
--- a/test/stdtypes/stdtypespb_test.go
+++ b/test/stdtypes/stdtypespb_test.go
@@ -606,10 +606,6 @@ func TestStdTypesGoString(t *testing.T) {
 	if s1 != s2 {
 		t.Fatalf("GoString want %v got %v", s1, s2)
 	}
-	_, err := go_parser.ParseExpr(s1)
-	if err != nil {
-		t.Fatal(err)
-	}
 }
 func TestRepStdTypesGoString(t *testing.T) {
 	popr := math_rand.New(math_rand.NewSource(time.Now().UnixNano()))
